src/main/java/com/onarandombox/MultiversePortals/PortalPlayerSession.java

```diff
@@ -257,24 +257,54 @@ public boolean showDebugInfo(MVPortal portal) {
     }
 
     private void showPortalPriceInfo(MVPortal portal) {
-        getPlayerFromName().sendMessage("More details for you: " + ChatColor.GREEN + portal.getDestination());
+        getPlayerFromName().sendMessage("More details: " + ChatColor.GREEN + portal.getDestination());
         if (portal.getPrice() > 0D) {
             getPlayerFromName().sendMessage("Price: " + ChatColor.GREEN + plugin.getCore().getEconomist().formatPrice(portal.getPrice(), portal.getCurrency()));
         } else if (portal.getPrice() < 0D) {
-            getPlayerFromName().sendMessage("Prize: " + ChatColor.GREEN + plugin.getCore().getEconomist().formatPrice(-portal.getPrice(), portal.getCurrency()));
+            getPlayerFromName().sendMessage("Price: " + ChatColor.GREEN + plugin.getCore().getEconomist().formatPrice(-portal.getPrice(), portal.getCurrency()));
         } else {
             getPlayerFromName().sendMessage("Price: " + ChatColor.GREEN + "FREE!");
         }
     }
 
     public static void showStaticInfo(CommandSender sender, MVPortal portal, String message) {
-        sender.sendMessage(message + ChatColor.DARK_AQUA + portal.getName());
-        sender.sendMessage("It's coords are: " + ChatColor.GOLD + portal.getLocation().toString());
+        sender.sendMessage(ChatColor.AQUA + "--- " + message + ChatColor.DARK_AQUA + portal.getName() + ChatColor.AQUA + " ---");
+        String[] locParts = portal.getLocation().toString().split(":");
+        sender.sendMessage("Coords from " + ChatColor.GOLD + locParts[0] + ChatColor.WHITE + " to " + ChatColor.GOLD + locParts[1] + ChatColor.WHITE + " in " + ChatColor.GOLD + portal.getWorld().getName() );
```

Comment:
Maybe we should do `Coords: `, instead of `Coords from ` to alight with how the rest of the new output looks.